marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FixCreatedByStepTest.java

```diff
@@ -179,8 +179,9 @@ private void verifyCreatedByStepIsFixed(String expectedValue) {
     private void addAnotherJobIdToSomeDocuments() {
         for (int i = 1; i <= 10; i++) {
             String script = "declareUpdate(); const uri = '/echo/customer" + i + ".json'; " +
+                "if (fn.docAvailable(uri)) {" +
```

Comment:
Do we know why some might not be available?

Reply:
All of them should be available because this test added them a some lines before,  but the thing is that one timeout is happening sometimes, for this reason I also reduced the number of records. To check if with these changes solve the problem.

Reply:
The check for docAvailable should also give us better messaging if the test fails so we can see if the counts are off and by how much they are off in the assertions.